tests/utils/mod.rs

```diff
@@ -14,10 +15,10 @@ pub use common::initialize_system;
 pub use fungible_token::FungibleToken;
 pub use non_fungible_token::NonFungibleToken;
 
-pub const FOREIGN_USER: u64 = 1029384756123;
-pub const PRODUCER: u64 = 5;
-pub const DISTRIBUTOR: u64 = 7;
-pub const RETAILER: u64 = 9;
+pub static FOREIGN_USER: u64 = 1029384756123;
```

Comment:
It no much sense to make `Copy` types `static`, I guess.

Reply:
I prefer `static` because it's not copied & pasted like `const` but is saved once in the static memory & used by reference.

Reply:
Using `const` is preferrable.

https://doc.rust-lang.org/reference/items/static-items.html#using-statics-or-consts